### PR TITLE
Add information on targets available in regex tolerances

### PR DESCRIPTION
**Pull request by ojongerius**

### CODE REVIEW

File: sources/reference/tutorials/tolerance.md [hunk 1]
```diff
@@ -153,11 +153,22 @@ On top of this native types, we support also more advance cases such as:
...
+    [Python engine][re]. This is useful when looking for a value in a raw string.
+    * `target` is optional, and allows changing the default target for a given provider.
+    
+    Currently supported targets per provider are as follows:
+    
+    | Provider | Default | Values |
+    | -------- | ------- | ------ |
+    | process  | `"status"` | `"stdout"`, `"stderr"` |
+    | http | `"status"` | `"headers"`, `"body"` |
+    | python | Undefined | Undefined |
```
Comment (ojongerius) [inline sources/reference/tutorials/tolerance.md:170]: Not sure how this would be phrased in a nicer way

**Comment by ojongerius** [comment]
Ping! @Lawouach?

**Comment by Lawouach** [comment]
Godh, I really need to set a slack integration to get pinged :)

I totally missed it :(

**Comment by Lawouach** [comment]
This is a good addition!

**Pull request merged**